pubsub/requests.go

```diff
@@ -35,3 +44,25 @@ type NewMessage struct {
 	Metadata    map[string]string `json:"metadata"`
 	ContentType *string           `json:"contentType,omitempty"`
 }
+
+// Represents batch request struct
```

Comment:
nit: first line of comment to contain the type name https://go.dev/doc/comment, missing comment for `BatchSubscribeConfig`

Reply:
https://go.dev/doc/comment is helpful link.